spec/services/invoices/advance_charges_service_spec.rb

```diff
@@ -160,6 +160,7 @@ def fee_boundaries
         create(
           :fee,
           :succeeded,
+          succeeded_at: fee_boundaries[:charges_to_datetime] - 2.days,
```

Comment:
Nice! 👏

Reply:
@vincent-pochet why `-2.days`? Is it arbitrary or does it have a logic behind it?